samples/SkiaMultiplatformSample/src/commonMain/kotlin/org/jetbrains/skiko/sample/Clocks.kt

```diff
@@ -7,45 +7,60 @@ import org.jetbrains.skia.paragraph.ParagraphStyle
 import org.jetbrains.skia.paragraph.TextStyle
 import org.jetbrains.skiko.SkikoView
 import org.jetbrains.skiko.SkikoPointerEvent
+import org.jetbrains.skiko.SkikoGestureEvent
+import org.jetbrains.skiko.SkikoGestureEventKind
+import org.jetbrains.skiko.SkikoGestureEventState
 import org.jetbrains.skiko.isLeftClick
 import kotlin.math.cos
 import kotlin.math.sin
 import kotlin.math.PI
 
 class Clocks: SkikoView {
     private var frame = 0
-    private var xpos = 0.0
-    private var ypos = 0.0
+    private var xpos = 0.0f
```

Comment:
Why Float? We use Doubles in events it seems.

Reply:
In events - yes, but Clock sample does not need such precision. This was done to avoid the frequent use of `toFloat ()`.